tests/api_tests/test_participant_api.py

```diff
@@ -662,6 +662,12 @@ def test_first_ehr_consent_not_set_on_no(self):
         participant_summary = self.send_get("Participant/%s/Summary" % participant_id)
         self.assertNotIn('consentForElectronicHealthRecordsFirstYesAuthored', participant_summary)
 
+    def test_pid_rid_mapping_api(self):
+        self.send_post("Participant", {"providerLink": [self.provider_link_2]})
+        result = self.send_get("ParticipantId/ResearchId/Mapping?signUpAfter=2020-01-01&sort=lastModified")
+        self.assertEqual(len(result['data']), 1)
+        self.assertEqual(result['sort_by'], 'lastModified')
```

Comment:
What do you think of checking that the response has the research_id field, and that it's correct?

Reply:
Done